### ISSUE TITLE
Remove billingPlan issue

### ISSUE THREAD

**Issue by jchapelle** [paypal/PayPal-node-SDK#151]
I'm trying to remove a plan.
The request seems to be accepted as I receive a statusCode 200 in the cb of the sdk.

But if I list my plans, the plan is still there ...

```
var billingPlanUpdateAttributes = [
                {
                    "op": "remove",
                    "path": "/"
                }
            ];

paypal.billingPlan.update(billingPlanId, billingPlanUpdateAttributes, function (error, billingPlan) 
...
```

**Comment by bluk** [comment]
The update operation is intended to update specific attributes of a plan (e.g. not delete it). I don't believe there is an operation that allows you to delete a billing plan due to the possibility of dependencies (e.g. billing agreements).

**Comment by jchapelle** [comment]
Ok thanks

On Thu, Apr 28, 2016 at 5:39 PM, bluk <notifications@github.com> wrote:

> Closed #151 <https://github.com/paypal/PayPal-node-SDK/issues/151>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/paypal/PayPal-node-SDK/issues/151#event-644495393>
>



-- 
Jean Chapelle

**Comment by skiod** [comment]
@bluk but what if we need to delete it ?? how could we do that ??? i didn't find anything related to delete plan
